java/inflector-petstore-sample/README.MD

```diff
@@ -0,0 +1,15 @@
+Inflector Jetty WebXML Sample
```

Comment:
what is this project? leftover to be removed?

Reply:
Removed! It was a sample project. But now everything is in swagger-petstore.